pallets/funding/src/benchmarking.rs

```diff
@@ -854,11 +863,22 @@ mod benchmarks {
 		let bidder = account::<AccountIdOf<T>>("bidder", 0, 0);
 		whitelist_account!(bidder);
 
-		let project_metadata = default_project::<T>(inst.get_new_nonce(), issuer.clone());
+		let mut project_metadata = default_project::<T>(inst.get_new_nonce(), issuer.clone());
+		project_metadata.mainnet_token_max_supply =
+			(1_000_000_000 * ASSET_UNIT).try_into().unwrap_or_else(|_| panic!("Failed to create BalanceOf"));
```

Comment:
Just general question, can we use 'expect' here? Or does that not work with some no_std functionality?

Reply:
Unfortunately not because expect still outputs the failed item which needs to implement debug:
![CleanShot 2024-04-11 at 10 08 39](https://github.com/Polimec/polimec-node/assets/54085674/e67e157d-1709-46ba-9fa2-6583f184e7a8)

Reply:
But if you have some other idiomatic way to unwrap items that don't implement `Debug` please let me know!

Reply:
https://stackoverflow.com/a/75345113/24201002 Maybe via let ... else?